nolearn/lasagne/base.py

```diff
@@ -584,20 +596,17 @@ def train_loop(self, X, y, epochs=None):
 
             batch_train_sizes = []
             for Xb, yb in self.batch_iterator_train(X_train, y_train):
-                batch_train_loss = self.apply_batch_func(
-                    self.train_iter_, Xb, yb)
-                train_losses.append(batch_train_loss[0])
```

Comment:
What did `batch_train_loss[0]` represent before? Now it looks like you are calling np.mean on the entire returned array ?